<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2 scRNA-seq介绍 | 生物信息学项目驱动实战</title>
  <meta name="description" content="2 scRNA-seq介绍 | 生物信息学项目驱动实战" />
  <meta name="generator" content="bookdown 0.11 and GitBook 2.6.7" />

  <meta property="og:title" content="2 scRNA-seq介绍 | 生物信息学项目驱动实战" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2 scRNA-seq介绍 | 生物信息学项目驱动实战" />
  
  
  

<meta name="author" content="碱基吃柠檬" />


<meta name="date" content="2019-07-04" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<!-- for Facebook -->  
<meta property="og:url" content="https://github.com/sdbiodog" />
<meta property="og:description" content="本项目是将github上经典的scRNA-seq分析教程进行汉化，源项目参见hemberg-lab/scRNA.seq.course" />
<meta property="og:image" content="http://hemberg-lab.github.io/scRNA.seq.course/figures/RNA-Seq_workflow-5.pdf.jpg" />

<!-- for Twitter -->          
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:title" content="scRNA-seq数据分析" />
<meta name="twitter:description" content="本项目是将github上经典的scRNA-seq分析教程进行汉化，源项目参见hemberg-lab/scRNA.seq.course" />
<meta name="twitter:image" content="http://hemberg-lab.github.io/scRNA.seq.course/figures/RNA-Seq_workflow-5.pdf.jpg" />

<!-- Google Analytics -->
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-71525309-1', 'auto');
  ga('send', 'pageview');

</script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="index.html">Table of Contents</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> 说在前面的话</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#section-1.1"><i class="fa fa-check"></i><b>1.1</b> 关于本项目</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#section-1.2"><i class="fa fa-check"></i><b>1.2</b> 许可</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#section-1.3"><i class="fa fa-check"></i><b>1.3</b> 准备知识</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#section-1.4"><i class="fa fa-check"></i><b>1.4</b> 联系我们</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html"><i class="fa fa-check"></i><b>2</b> scRNA-seq介绍</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#bulk-rna-seq"><i class="fa fa-check"></i><b>2.1</b> Bulk RNA-seq</a></li>
<li class="chapter" data-level="2.2" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#scrna-seq"><i class="fa fa-check"></i><b>2.2</b> scRNA-seq</a></li>
<li class="chapter" data-level="2.3" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#section-2.3"><i class="fa fa-check"></i><b>2.3</b> 工作流程</a></li>
<li class="chapter" data-level="2.4" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#section-2.4"><i class="fa fa-check"></i><b>2.4</b> 计算分析</a></li>
<li class="chapter" data-level="2.5" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#section-2.5"><i class="fa fa-check"></i><b>2.5</b> 挑战</a></li>
<li class="chapter" data-level="2.6" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#section-2.6"><i class="fa fa-check"></i><b>2.6</b> 实验方法</a></li>
<li class="chapter" data-level="2.7" data-path="introduction-to-single-cell-rna-seq.html"><a href="introduction-to-single-cell-rna-seq.html#section-2.7"><i class="fa fa-check"></i><b>2.7</b> 如何选择合适的平台</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/lemonbases/scRNA.seq.course.CN" target="blank">碱基吃柠檬</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">生物信息学项目驱动实战</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction_to_single_cell_rna_seq" class="section level1">
<h1><span class="header-section-number">2</span> scRNA-seq介绍</h1>
<div id="bulk-rna-seq" class="section level2">
<h2><span class="header-section-number">2.1</span> Bulk RNA-seq</h2>
<ul>
<li>2000后期重大突破，取代微阵列芯片被广泛采用</li>
<li>测量大量混合细胞的<strong>平均表达水平</strong></li>
<li>用于比较转录组学，例如比较不同物种同一组织的样本</li>
<li>用于量化整体表达特征，例如疾病研究</li>
<li><strong>不足</strong>研究异质性系统，例如早起发育的研究，复杂组织(脑)的研究</li>
<li><strong>不能</strong>呈现基因表达随机性</li>
</ul>
</div>
<div id="scrna-seq" class="section level2">
<h2><span class="header-section-number">2.2</span> scRNA-seq</h2>
<ul>
<li>一项<strong>新兴</strong>技术，第一篇文章为汤富酬发表在Nature method <span class="citation">(Tang et al. <a href="#ref-Tang2009-bu">2009</a>)</span></li>
<li>直到<a href="https://www.ohio.edu/bioinformatics/upload/Single-Cell-RNA-seq-Method-of-the-Year-2013.pdf">~2014</a>年，新测序手段和更低的成本使其流行起来</li>
<li>在大量细胞检测每个基因的表达水平</li>
<li>可以研究新的生物学问题，其中<strong>转录组的细胞特异性变化是重要的</strong>，比如细胞类型鉴定，细胞响应的异质性，基因表达的随机性，细胞间基因调控网络的推断</li>
<li>研究细胞数目从<strong><span class="math inline">\(10^2\)</span></strong>到<em><span class="math inline">\(10^6\)</span></em>个细胞，而且每年递增</li>
<li>目前有不同单细胞的protocol，比如SMART-seq2 <span class="citation">(Picelli et al. <a href="#ref-Picelli2013-sb">2013</a>)</span>, CELL-seq <span class="citation">(Hashimshony et al. <a href="#ref-Hashimshony2012-kd">2012</a>)</span> 和 Drop-seq <span class="citation">(Macosko et al. <a href="#ref-Macosko2015-ix">2015</a>)</span></li>
<li>也有商业平台, 包括 <a href="https://www.fluidigm.com/products/c1-system">Fluidigm C1</a>, <a href="https://www.wafergen.com/products/icell8-single-cell-system">Wafergen ICELL8</a> 和 <a href="https://www.10xgenomics.com/single-cell/">10X Genomics Chromium</a></li>
<li>一些计算分析bulk RNA-seq的方法<strong>也适用于</strong> scRNA-seq</li>
<li><strong>大多数情况下</strong> scRNA-seq计算分析需要调整已有方法或者开发新方法</li>
</ul>
</div>
<div id="section-2.3" class="section level2">
<h2><span class="header-section-number">2.3</span> 工作流程</h2>
<div class="figure" style="text-align: center"><span id="fig:intro-rna-seq-workflow"></span>
<img src="figures/RNA-Seq_workflow-5.pdf.jpg" alt="单细胞测序流程 (源自wiki)" width="90%" />
<p class="caption">
Figure 2.1: 单细胞测序流程 (源自wiki)
</p>
</div>
<p>总体而言，scRNA-seq实验protocol和bulk RNA-seq类似，我们将在下一章讨论一些最常用的方法。</p>
</div>
<div id="section-2.4" class="section level2">
<h2><span class="header-section-number">2.4</span> 计算分析</h2>
<p>本课程是用scRNA-seq实验获得的数据进行计算分析。</p>
<ol style="list-style-type: decimal">
<li>第一步（黄色）对任何高通量测序数据都是通用的；</li>
<li>后续步骤（橙色）需要整合现有的RNA-Seq分析方法和新方法来解决scRNA-Seq的技术差异；</li>
<li>最后步骤（蓝色），用专门为scRNA-Seq开发的方法进行生物学解释。</li>
</ol>
<div class="figure" style="text-align: center"><span id="fig:intro-flowchart"></span>
<img src="figures/flowchart.png" alt="scRNA-seq分析流程图" width="65%" />
<p class="caption">
Figure 2.2: scRNA-seq分析流程图
</p>
</div>
<p>目前有几篇关于scRNA-seq分析的综述文章，包括 <span class="citation">(Stegle, Teichmann, and Marioni <a href="#ref-Stegle2015-uv">2015</a>)</span></p>
<p>目前，也有不同的平台执行上述流程图的一步或多个步骤，包括:</p>
<ul>
<li><a href="https://github.com/VCCRI/Falco/">Falco</a> 云端单细胞RNA-seq处理框架</li>
<li><a href="https://github.com/YosefLab/scone">SCONE</a> (Single-Cell Overview of Normalized Expression), 单细胞RNA-seq数据质量控制和标准化的包.</li>
<li><a href="http://satijalab.org/seurat/">Seurat</a> 用于单细胞RNA-seq数据质量控制，分析和数据探索的R包.</li>
<li><a href="https://asap.epfl.ch/">ASAP</a> (Automated Single-cell Analysis Pipeline) 是一个单细胞分析交互式webserver.</li>
<li><a href="https://master.bioconductor.org/packages/release/workflows/html/simpleSingleCell.html">Bioconductor</a> 是一个分析高通量基因组数据开源，开放式软件项目，包括分析单细胞数据的工具包。</li>
</ul>
</div>
<div id="section-2.5" class="section level2">
<h2><span class="header-section-number">2.5</span> 挑战</h2>
<p>scRNA-seq和bulk RNA-seq最大的不同在于每个测序文库是<strong>单个细胞</strong>，而不是一群细胞。因此需要各位注意来自不同细胞（测序文库）的结果的比较。文库之间差异的主要来源是:</p>
<ul>
<li><strong>扩增偏好性和扩增效率</strong> （高达1,000,000倍）</li>
<li><strong>基因的’dropout’</strong>，其中一个基因在一个细胞中等表达水平，但在另外一个细胞没有检测到<span class="citation">(Kharchenko, Silberstein, and Scadden <a href="#ref-Kharchenko2014-ts">2014</a>)</span>.</li>
</ul>
<p>在两种情况下， 由于RNA仅来自一个细胞，因此低起始量的转录本是导致差异的主要原因。提高转录本捕获效率和减少扩增偏差是目前活跃的研究灵越。从后续的课程我们也可以看到，可以通过适当的标准化和校正方法来缓解这些问题。</p>
</div>
<div id="section-2.6" class="section level2">
<h2><span class="header-section-number">2.6</span> 实验方法</h2>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-2"></span>
<img src="figures/moores-law.png" alt="单细胞转录组学摩尔定律 (图片来自 [Svensson et al](https://arxiv.org/abs/1704.01379))" width="100%" />
<p class="caption">
Figure 2.3: 单细胞转录组学摩尔定律 (图片来自 <a href="https://arxiv.org/abs/1704.01379">Svensson et al</a>)
</p>
</div>
<p>开发新的scRNA-seq方法和protocol是目前非常活跃的一个研究领域，在过去的几年中已经发表了一些protocol，不完全列表如下:</p>
<ul>
<li>CEL-seq <span class="citation">(Hashimshony et al. <a href="#ref-Hashimshony2012-kd">2012</a>)</span></li>
<li>CEL-seq2 <span class="citation">(Hashimshony et al. <a href="#ref-Hashimshony2016-lx">2016</a>)</span></li>
<li>Drop-seq <span class="citation">(Macosko et al. <a href="#ref-Macosko2015-ix">2015</a>)</span></li>
<li>InDrop-seq <span class="citation">(Klein et al. <a href="#ref-Klein2015-kz">2015</a>)</span></li>
<li>MARS-seq <span class="citation">(Jaitin et al. <a href="#ref-Jaitin2014-ko">2014</a>)</span></li>
<li>SCRB-seq <span class="citation">(Soumillon et al. <a href="#ref-Soumillon2014-eu">2014</a>)</span></li>
<li>Seq-well <span class="citation">(Gierahn et al. <a href="#ref-Gierahn2017-es">2017</a>)</span></li>
<li>Smart-seq <span class="citation">(Picelli et al. <a href="#ref-Picelli2014-ic">2014</a>)</span></li>
<li>Smart-seq2 <span class="citation">(Picelli et al. <a href="#ref-Picelli2014-ic">2014</a>)</span></li>
<li><a href="http://www.clontech.com/US/Products/cDNA_Synthesis_and_Library_Construction/Next_Gen_Sequencing_Kits/Total_RNA-Seq/Universal_RNA_Seq_Random_Primed">SMARTer</a></li>
<li>STRT-seq <span class="citation">(Islam et al. <a href="#ref-Islam2014-cn">2013</a>)</span></li>
</ul>
<p>这些方法可以按照不同方法进行归类，最重要的两个方面是<strong>定量</strong>和<strong>捕获</strong></p>
<p>定量有两种类型，<strong>全长</strong>和<strong>基于标签</strong>。前者试图达到每个转录本均匀覆盖率；然而基于tag的protocol只捕获每个RNA的5’或3’端。量化方法的选择对于数据可用于何种类型的分析具有重要意义。理论上讲，基于全长的protocol可以对整个转录本进行均匀测序，然而通常测序覆盖有偏差。基于tag的protocol主要优势是其可以和唯一的分子标识符结合使用，提高定量准确性(see chapter <a href="#umichapter"><strong>??</strong></a>)。另一方面，测序限制在转录组的一端，降低比对map率，并难以区分不同的异构体<span class="citation">(Archer et al. <a href="#ref-Archer2016-zq">2016</a>)</span>。</p>
<p>捕获的策略决定了通量，细胞如何被选择以及除测序外还可以获得哪种附加信息。三种常用的方法是基于<strong>微孔(microwell)-</strong>，<strong>微流体(microfluidic)-</strong>，<strong>液滴(droplet)-</strong>的方法。</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-3"></span>
<img src="figures/300px-Microplates.jpg" alt="微孔板 (图片来自wiki)" width="150" />
<p class="caption">
Figure 2.4: 微孔板 (图片来自wiki)
</p>
</div>
<p>对于基于微孔的平台，使用例如移液管或激光捕获分离细胞并置于微流体孔中。 基于微孔的方法的一个优点是它们可以与荧光激活细胞分选（FACS）结合，基于表面标记物分选细胞。 因此，当想要分离特定细胞群体用于测序时，该策略非常有用。 另一个优点是可以拍摄细胞的照片。 该图像提供了额外的细胞形态，适用于识别包含受损细胞或双份细胞的微孔。 这些方法的主要缺点是它们通常是通量低并且每个单细胞所需的工作量可能相当大。</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-4"></span>
<img src="figures/fluidigmC1.jpg" alt="96孔Fluidigm C1芯片 (图片来自Fluidigm)" width="350" />
<p class="caption">
Figure 2.5: 96孔Fluidigm C1芯片 (图片来自Fluidigm)
</p>
</div>
<p>微流体平台, 比如 <a href="https://www.fluidigm.com/products/c1-system#workflow">Fluidigm’s C1</a>, 提供了更加集成的系统，用于捕获细胞和进行文库制备所需的过程。 因此，它们提供比基于微孔的平台更高的通量。 通常，在微流体平台中仅捕获约10％的细胞，因此不适合处理稀有细胞类型或非常少量的细胞。 此外，芯片相对昂贵，但由于反应可以以较小的体积进行，因此可以节省试剂。</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-5"></span>
<img src="figures/drop-seq.png" alt="drop-seq方法的原理图 (图片来自Macosko et al)" width="60%" />
<p class="caption">
Figure 2.6: drop-seq方法的原理图 (图片来自Macosko et al)
</p>
</div>
<p>基于液滴的方法将每个单独的细胞与包括建库所需酶的珠子(bead)一起封装在纳升液滴内。每个珠子都包含唯一的条形码(barcode)，加到所有来自该细胞的序列上。因此可以合并所有液滴进行测序，再基于barcode将序列分配给不同的细胞。Droplet平台通常具有最高的通量，因为建库准备成本约为<span class="math inline">\(.05\)</span> 美元/细胞。相反，测序成本成为其限制因素，通常实验覆盖率低，仅检测到几千个转录本 <span class="citation">(Ziegenhain et al. <a href="#ref-Ziegenhain2017-cu">2017</a>)</span>。</p>
</div>
<div id="section-2.7" class="section level2">
<h2><span class="header-section-number">2.7</span> 如何选择合适的平台</h2>
<p>最合适的平台取决于手头的生物学问题。 例如，如果研究组织的组成，那么将允许捕获非常大量细胞的基于液滴的方法可能是最合适的。 另一方面，如果人们对具有已知表面标记物的稀有细胞类群感兴趣，那么最好使用FACS进行富集，然后对较少数量的细胞进行测序。</p>
<p>显然，如果研究不同的异构体，那么全长转录物定量将更合适。相比之下，UMI只能与基于tag的protocol一起使用，促进基因水平的量化。</p>
<p>来自Enard group <span class="citation">(Ziegenhain et al. <a href="#ref-Ziegenhain2017-cu">2017</a>)</span>和Teichmann group <span class="citation">(Svensson et al. <a href="#ref-Svensson2017-op">2017</a>)</span>最近两项研究比较了几种不同的protocol。Ziegenhain等人在同一小鼠胚胎干细胞样本（mESCs）上比较了五种不同的protocol。通过控制细胞数量和测序深度，作者能够直接比较不同protocol的灵敏度，噪声水平和成本。 他们的结论的一个例子如下图所示，不同protocol检测的基因数量（对于给定的检测阈值）差别很大。drop-seq和Smart-seq2之间几乎有两倍的差异，这表明protocol的选择会对研究有重大影响。</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-6"></span>
<img src="figures/ziegenhainEnardFig1.png" alt="Enard group 研究" width="60%" />
<p class="caption">
Figure 2.7: Enard group 研究
</p>
</div>
<p>Svensson等采用了另外一种方法，通过使用已知浓度的合成转录本（<strong>spike-ins</strong>，后面详细介绍）来测量不同protocol的准确性和灵敏度。通过广泛的比较研究，他们同样发现不同protocol间区别较大。</p>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-7"></span>
<img src="figures/svenssonTeichmannFig2.png" alt="Teichmann group 研究" width="100%" />
<p class="caption">
Figure 2.8: Teichmann group 研究
</p>
</div>
<p>随着实验技术的发展和用于量化技术噪声的计算方法的改进，后续研究有助于我们进一步了解不同方法的优缺点。因为基准测试可以确定哪些策略是最有用的，这些比较研究不仅有助于研究人员决定使用哪种protocol，而且有助于开发新方法。</p>

<div id="refs" class="references">
<div>
<p>Archer, Nathan, Mark D. Walsh, Vahid Shahrezaei, and Daniel Hebenstreit. 2016. “Modeling Enzyme Processivity Reveals That RNA-Seq Libraries Are Biased in Characteristic and Correctable Ways.” <em>Cell Systems</em> 3 (5). Elsevier BV: 467–479.e12. doi:<a href="https://doi.org/10.1016/j.cels.2016.10.012">10.1016/j.cels.2016.10.012</a>.</p>
</div>
<div>
<p>Gierahn, Todd M, Marc H Wadsworth 2nd, Travis K Hughes, Bryan D Bryson, Andrew Butler, Rahul Satija, Sarah Fortune, J Christopher Love, and Alex K Shalek. 2017. “Seq-Well: Portable, Low-Cost RNA Sequencing of Single Cells at High Throughput.” <em>Nat. Methods</em> 14 (4): 395–98.</p>
</div>
<div>
<p>Hashimshony, Tamar, Naftalie Senderovich, Gal Avital, Agnes Klochendler, Yaron de Leeuw, Leon Anavy, Dave Gennert, et al. 2016. “CEL-Seq2: Sensitive Highly-Multiplexed Single-Cell RNA-Seq.” <em>Genome Biol</em> 17 (1). Springer Nature. doi:<a href="https://doi.org/10.1186/s13059-016-0938-8">10.1186/s13059-016-0938-8</a>.</p>
</div>
<div>
<p>Hashimshony, Tamar, Florian Wagner, Noa Sher, and Itai Yanai. 2012. “CEL-Seq: Single-cell RNA-Seq by Multiplexed Linear Amplification.” <em>Cell Reports</em> 2 (3). Elsevier BV: 666–73. doi:<a href="https://doi.org/10.1016/j.celrep.2012.08.003">10.1016/j.celrep.2012.08.003</a>.</p>
</div>
<div>
<p>Islam, Saiful, Amit Zeisel, Simon Joost, Gioele La Manno, Pawel Zajac, Maria Kasper, Peter Lönnerberg, and Sten Linnarsson. 2013. “Quantitative Single-Cell RNA-Seq with Unique Molecular Identifiers.” <em>Nat Meth</em> 11 (2). Springer Nature: 163–66. doi:<a href="https://doi.org/10.1038/nmeth.2772">10.1038/nmeth.2772</a>.</p>
</div>
<div>
<p>Jaitin, Diego Adhemar, Ephraim Kenigsberg, Hadas Keren-Shaul, Naama Elefant, Franziska Paul, Irina Zaretsky, Alexander Mildner, et al. 2014. “Massively Parallel Single-Cell RNA-seq for Marker-Free Decomposition of Tissues into Cell Types.” <em>Science</em> 343 (6172): 776–79.</p>
</div>
<div>
<p>Kharchenko, Peter V, Lev Silberstein, and David T Scadden. 2014. “Bayesian Approach to Single-Cell Differential Expression Analysis.” <em>Nat Meth</em> 11 (7). Springer Nature: 740–42. doi:<a href="https://doi.org/10.1038/nmeth.2967">10.1038/nmeth.2967</a>.</p>
</div>
<div>
<p>Klein, Allon M., Linas Mazutis, Ilke Akartuna, Naren Tallapragada, Adrian Veres, Victor Li, Leonid Peshkin, David A. Weitz, and Marc W. Kirschner. 2015. “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells.” <em>Cell</em> 161 (5). Elsevier BV: 1187–1201. doi:<a href="https://doi.org/10.1016/j.cell.2015.04.044">10.1016/j.cell.2015.04.044</a>.</p>
</div>
<div>
<p>Macosko, Evan Z., Anindita Basu, Rahul Satija, James Nemesh, Karthik Shekhar, Melissa Goldman, Itay Tirosh, et al. 2015. “Highly Parallel Genome-Wide Expression Profiling of Individual Cells Using Nanoliter Droplets.” <em>Cell</em> 161 (5). Elsevier BV: 1202–14. doi:<a href="https://doi.org/10.1016/j.cell.2015.05.002">10.1016/j.cell.2015.05.002</a>.</p>
</div>
<div>
<p>Picelli, Simone, Åsa K Björklund, Omid R Faridani, Sven Sagasser, Gösta Winberg, and Rickard Sandberg. 2013. “Smart-Seq2 for Sensitive Full-Length Transcriptome Profiling in Single Cells.” <em>Nat Meth</em> 10 (11). Springer Nature: 1096–8. doi:<a href="https://doi.org/10.1038/nmeth.2639">10.1038/nmeth.2639</a>.</p>
</div>
<div>
<p>Picelli, Simone, Omid R Faridani, Asa K Björklund, Gösta Winberg, Sven Sagasser, and Rickard Sandberg. 2014. “Full-Length RNA-seq from Single Cells Using Smart-Seq2.” <em>Nat. Protoc.</em> 9 (1): 171–81.</p>
</div>
<div>
<p>Soumillon, Magali, Davide Cacchiarelli, Stefan Semrau, Alexander van Oudenaarden, and Tarjei S Mikkelsen. 2014. “Characterization of Directed Differentiation by High-Throughput Single-Cell RNA-Seq.” <em>bioRxiv</em>, March, 003236.</p>
</div>
<div>
<p>Stegle, Oliver, Sarah A. Teichmann, and John C. Marioni. 2015. “Computational and Analytical Challenges in Single-Cell Transcriptomics.” <em>Nat Rev Genet</em> 16 (3). Springer Nature: 133–45. doi:<a href="https://doi.org/10.1038/nrg3833">10.1038/nrg3833</a>.</p>
</div>
<div>
<p>Svensson, Valentine, Kedar Nath Natarajan, Lam-Ha Ly, Ricardo J Miragaia, Charlotte Labalette, Iain C Macaulay, Ana Cvejic, and Sarah A Teichmann. 2017. “Power Analysis of Single-Cell RNA-Sequencing Experiments.” <em>Nat Meth</em> 14 (4). Springer Nature: 381–87. doi:<a href="https://doi.org/10.1038/nmeth.4220">10.1038/nmeth.4220</a>.</p>
</div>
<div>
<p>Tang, Fuchou, Catalin Barbacioru, Yangzhou Wang, Ellen Nordman, Clarence Lee, Nanlan Xu, Xiaohui Wang, et al. 2009. “mRNA-Seq Whole-Transcriptome Analysis of a Single Cell.” <em>Nat Meth</em> 6 (5). Springer Nature: 377–82. doi:<a href="https://doi.org/10.1038/nmeth.1315">10.1038/nmeth.1315</a>.</p>
</div>
<div>
<p>Ziegenhain, Christoph, Beate Vieth, Swati Parekh, Björn Reinius, Amy Guillaumet-Adkins, Martha Smets, Heinrich Leonhardt, Holger Heyn, Ines Hellmann, and Wolfgang Enard. 2017. “Comparative Analysis of Single-Cell RNA Sequencing Methods.” <em>Molecular Cell</em> 65 (4). Elsevier BV: 631–643.e4. doi:<a href="https://doi.org/10.1016/j.molcel.2017.01.023">10.1016/j.molcel.2017.01.023</a>.</p>
</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Archer2016-zq">
<p>Archer, Nathan, Mark D. Walsh, Vahid Shahrezaei, and Daniel Hebenstreit. 2016. “Modeling Enzyme Processivity Reveals That RNA-Seq Libraries Are Biased in Characteristic and Correctable Ways.” <em>Cell Systems</em> 3 (5). Elsevier BV: 467–479.e12. doi:<a href="https://doi.org/10.1016/j.cels.2016.10.012">10.1016/j.cels.2016.10.012</a>.</p>
</div>
<div id="ref-Gierahn2017-es">
<p>Gierahn, Todd M, Marc H Wadsworth 2nd, Travis K Hughes, Bryan D Bryson, Andrew Butler, Rahul Satija, Sarah Fortune, J Christopher Love, and Alex K Shalek. 2017. “Seq-Well: Portable, Low-Cost RNA Sequencing of Single Cells at High Throughput.” <em>Nat. Methods</em> 14 (4): 395–98.</p>
</div>
<div id="ref-Hashimshony2016-lx">
<p>Hashimshony, Tamar, Naftalie Senderovich, Gal Avital, Agnes Klochendler, Yaron de Leeuw, Leon Anavy, Dave Gennert, et al. 2016. “CEL-Seq2: Sensitive Highly-Multiplexed Single-Cell RNA-Seq.” <em>Genome Biol</em> 17 (1). Springer Nature. doi:<a href="https://doi.org/10.1186/s13059-016-0938-8">10.1186/s13059-016-0938-8</a>.</p>
</div>
<div id="ref-Hashimshony2012-kd">
<p>Hashimshony, Tamar, Florian Wagner, Noa Sher, and Itai Yanai. 2012. “CEL-Seq: Single-cell RNA-Seq by Multiplexed Linear Amplification.” <em>Cell Reports</em> 2 (3). Elsevier BV: 666–73. doi:<a href="https://doi.org/10.1016/j.celrep.2012.08.003">10.1016/j.celrep.2012.08.003</a>.</p>
</div>
<div id="ref-Islam2014-cn">
<p>Islam, Saiful, Amit Zeisel, Simon Joost, Gioele La Manno, Pawel Zajac, Maria Kasper, Peter Lönnerberg, and Sten Linnarsson. 2013. “Quantitative Single-Cell RNA-Seq with Unique Molecular Identifiers.” <em>Nat Meth</em> 11 (2). Springer Nature: 163–66. doi:<a href="https://doi.org/10.1038/nmeth.2772">10.1038/nmeth.2772</a>.</p>
</div>
<div id="ref-Jaitin2014-ko">
<p>Jaitin, Diego Adhemar, Ephraim Kenigsberg, Hadas Keren-Shaul, Naama Elefant, Franziska Paul, Irina Zaretsky, Alexander Mildner, et al. 2014. “Massively Parallel Single-Cell RNA-seq for Marker-Free Decomposition of Tissues into Cell Types.” <em>Science</em> 343 (6172): 776–79.</p>
</div>
<div id="ref-Kharchenko2014-ts">
<p>Kharchenko, Peter V, Lev Silberstein, and David T Scadden. 2014. “Bayesian Approach to Single-Cell Differential Expression Analysis.” <em>Nat Meth</em> 11 (7). Springer Nature: 740–42. doi:<a href="https://doi.org/10.1038/nmeth.2967">10.1038/nmeth.2967</a>.</p>
</div>
<div id="ref-Klein2015-kz">
<p>Klein, Allon M., Linas Mazutis, Ilke Akartuna, Naren Tallapragada, Adrian Veres, Victor Li, Leonid Peshkin, David A. Weitz, and Marc W. Kirschner. 2015. “Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells.” <em>Cell</em> 161 (5). Elsevier BV: 1187–1201. doi:<a href="https://doi.org/10.1016/j.cell.2015.04.044">10.1016/j.cell.2015.04.044</a>.</p>
</div>
<div id="ref-Macosko2015-ix">
<p>Macosko, Evan Z., Anindita Basu, Rahul Satija, James Nemesh, Karthik Shekhar, Melissa Goldman, Itay Tirosh, et al. 2015. “Highly Parallel Genome-Wide Expression Profiling of Individual Cells Using Nanoliter Droplets.” <em>Cell</em> 161 (5). Elsevier BV: 1202–14. doi:<a href="https://doi.org/10.1016/j.cell.2015.05.002">10.1016/j.cell.2015.05.002</a>.</p>
</div>
<div id="ref-Picelli2013-sb">
<p>Picelli, Simone, Åsa K Björklund, Omid R Faridani, Sven Sagasser, Gösta Winberg, and Rickard Sandberg. 2013. “Smart-Seq2 for Sensitive Full-Length Transcriptome Profiling in Single Cells.” <em>Nat Meth</em> 10 (11). Springer Nature: 1096–8. doi:<a href="https://doi.org/10.1038/nmeth.2639">10.1038/nmeth.2639</a>.</p>
</div>
<div id="ref-Picelli2014-ic">
<p>Picelli, Simone, Omid R Faridani, Asa K Björklund, Gösta Winberg, Sven Sagasser, and Rickard Sandberg. 2014. “Full-Length RNA-seq from Single Cells Using Smart-Seq2.” <em>Nat. Protoc.</em> 9 (1): 171–81.</p>
</div>
<div id="ref-Soumillon2014-eu">
<p>Soumillon, Magali, Davide Cacchiarelli, Stefan Semrau, Alexander van Oudenaarden, and Tarjei S Mikkelsen. 2014. “Characterization of Directed Differentiation by High-Throughput Single-Cell RNA-Seq.” <em>bioRxiv</em>, March, 003236.</p>
</div>
<div id="ref-Stegle2015-uv">
<p>Stegle, Oliver, Sarah A. Teichmann, and John C. Marioni. 2015. “Computational and Analytical Challenges in Single-Cell Transcriptomics.” <em>Nat Rev Genet</em> 16 (3). Springer Nature: 133–45. doi:<a href="https://doi.org/10.1038/nrg3833">10.1038/nrg3833</a>.</p>
</div>
<div id="ref-Svensson2017-op">
<p>Svensson, Valentine, Kedar Nath Natarajan, Lam-Ha Ly, Ricardo J Miragaia, Charlotte Labalette, Iain C Macaulay, Ana Cvejic, and Sarah A Teichmann. 2017. “Power Analysis of Single-Cell RNA-Sequencing Experiments.” <em>Nat Meth</em> 14 (4). Springer Nature: 381–87. doi:<a href="https://doi.org/10.1038/nmeth.4220">10.1038/nmeth.4220</a>.</p>
</div>
<div id="ref-Tang2009-bu">
<p>Tang, Fuchou, Catalin Barbacioru, Yangzhou Wang, Ellen Nordman, Clarence Lee, Nanlan Xu, Xiaohui Wang, et al. 2009. “mRNA-Seq Whole-Transcriptome Analysis of a Single Cell.” <em>Nat Meth</em> 6 (5). Springer Nature: 377–82. doi:<a href="https://doi.org/10.1038/nmeth.1315">10.1038/nmeth.1315</a>.</p>
</div>
<div id="ref-Ziegenhain2017-cu">
<p>Ziegenhain, Christoph, Beate Vieth, Swati Parekh, Björn Reinius, Amy Guillaumet-Adkins, Martha Smets, Heinrich Leonhardt, Holger Heyn, Ines Hellmann, and Wolfgang Enard. 2017. “Comparative Analysis of Single-Cell RNA Sequencing Methods.” <em>Molecular Cell</em> 65 (4). Elsevier BV: 631–643.e4. doi:<a href="https://doi.org/10.1016/j.molcel.2017.01.023">10.1016/j.molcel.2017.01.023</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": true,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/lemonbases/scRNA.seq.course.CN/edit/master/02intro.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["scRNA-seq-course.pdf"],
"toc": {
"collapse": "section"
},
"search": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
